google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -1128,6 +1128,8 @@ def file path, generation: nil, skip_lookup: nil, encryption_key: nil
         # @param [String] content_type The
         #   [Content-Type](https://tools.ietf.org/html/rfc2616#section-14.17)
         #   response header to be returned when the file is downloaded.
+        # @param [DateTime] custom_time A custom time specified by the user for
+        #   the file.
```

Comment:
Once set it can't be unset and only changed to a custom datetime in the future. If the custom_time must be unset, you must either perform a rewrite operation or upload the data again.

Reply:
Done.